lua/weapons/arccw_base/cl_hud.lua

```diff
@@ -211,6 +216,9 @@ function SWEP:DrawHUD()
 
             if alpha > 0 then
 
+				local EyeAng = EyeAngles()
+				angpos.Pos = angpos.Pos - EyeAng:Up() * 5 - EyeAng:Right() * 4
```

Comment:
sometimes larger weapons will obscure it
and it can get in the way of what the player's tryna look at